src/search.c

```diff
@@ -341,7 +341,7 @@ static void print_help()
 	fprintf(stderr, "   -l, --library           Search paths where libraries are located for a match\n");
 	fprintf(stderr, "   -b, --binary            Search paths where binaries are located for a match\n");
 	fprintf(stderr, "   -s, --scope=[query type] 'b' or 'o' for first hit per (b)undle, or one hit total across the (o)s\n");
-	fprintf(stderr, "   -t, --top=[NUM]         Only display the top NUM results for each bundle\n");
+	fprintf(stderr, "   -T, --top=[NUM]         Only display the top NUM results for each bundle\n");
```

Comment:
Technically a breaking change to the interface, but since this is a not-often-scripted subcommand maybe we can get away with *not* bumping the MAJOR version according to semver. Thoughts?